src/styles/transformers/rtl.js

```diff
@@ -0,0 +1,90 @@
+import merge from '../../merge';
+import warning from 'warning';
+
+const reTranslate = /((^|\s)translate(3d|X)?\()(\-?[\d]+)/;
+const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(deg|rad|grad))?/;
+
+/**
+ * This function ensures that `style` supports both ltr and rtl directions by
+ * checking `styleConstants` in `muiTheme` and replacing attribute keys if
+ * necessary.
+ */
+export default function rtl(muiTheme, style) {
```

Comment:
Mm, actually for some reason I'm having trouble seeing how this could work: 

```js
export default theme => style => {
//...
};
```
If I'm not mistaken, since the `compose` function is taking a single return value and passing it as an argument to the next function in the chain, if we composed a set of functions with this signature, then wouldn't the second function in the chain receive the style in the theme?

I think the only way we can properly use compose with multiple objects such as theme and style, is to package them in an object together and make that the combined object the return and input values for the function.  Does that make sense?